actions/RestTestTakers.php

```diff
@@ -208,6 +213,15 @@ public function post()
         try {
             /** @var \core_kernel_classes_Resource $testTakerResource */
             $testTakerResource = parent::post();
+            $parameters = $this->getParameters();
+
+            /** @var tao_models_classes_UserService $userService */
+            $userService = $this->getServiceLocator()->get(tao_models_classes_UserService::SERVICE_ID);
+            $userService->triggerUpdatedEvent(
+                $testTakerResource,
+                [UserRdf::PROPERTY_PASSWORD => $testTakerResource->getProperty(UserRdf::PROPERTY_PASSWORD)],
+                isset($parameters['password']) ? UserHashForEncryption::hash($parameters['password']) : null
```

Comment:
Not sure why, but we use rdf properties names for API parameters. You have to check if `UserRdf::PROPERTY_PASSWORD` exists in $parameters instead of `password`

Reply:
There's no need, because `password` is required for TT API: [RestTestTakers.php#L187](https://github.com/oat-sa/extension-tao-testtaker/blob/cfc55cb7e3cbf47713724ece0d56be72dc8c8166/actions/RestTestTakers.php#L187)